setup.py

```diff
@@ -12,11 +12,12 @@
     "pyramid_jinja2",
     "pyramid_debugtoolbar",
     "waitress",
-    "legendarium==2.0.2",
-    "requests==2.21.0",
-    "articlemetaapi==1.26.5",
-    "picles.plumber==0.11",
-    "lxml==4.3.1",
+    "legendarium",
+    "requests",
+    "articlemetaapi>=1.26.6",
+    "picles.plumber",
```

Comment:
@cesarbruschetta aqui tem que ir com >=1.26.6?

Reply:
@robertatakenaka , sim foi o que o @gustavofonseca  sugeriu no comentario acima  

> Aqui há uma restrição para `articlemetaapi>=1.26.6`
_Originally posted by @gustavofonseca in https://github.com/scieloorg/document-store-migracao/pull/95#discussion_r283519329